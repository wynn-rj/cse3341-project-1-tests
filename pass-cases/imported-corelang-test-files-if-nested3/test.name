if nested3 (Imported from corelan-test-files)
