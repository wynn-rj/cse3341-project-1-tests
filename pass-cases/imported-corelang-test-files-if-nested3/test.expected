program
  int b;
begin
  b:=1;
  if b=1 then
    output 1;
    if b=2 then
      output 2;
    endif;
  else
    output 0;
  endif;
end
1
